### PR TITLE
MTV-465 document supported versions hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
@mavital @ahadas 
MTV-465:
Changed "Compatible software versions" table for MTV version 2.4.0.
Change supported OCP to 4.11 and 4.12. Please review and comment/correct if necessary.
Please review other supported software.

Thanks

### CODE REVIEW

File: documentation/modules/rn-2.4.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
```
Comment (ahadas) [inline documentation/modules/rn-2.4.adoc:8]: @RichardHoch we'd need to add openstack here as well ^

File: documentation/modules/rn-2.4.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+This release has the following resolved issue:
+
+.link:https://issues.redhat.com/browse/MTV-332[MTV-332] - Update virt-v2v container to el9
+
+* This fix enables conversion to RHEL9 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2067171#c1[BZ-2067171])
+* A fix in virt-v2v causes it to install quem-ga within the converted guest (link:https://bugzilla.redhat.com/show_bug.cgi?id=2028764[BZ-2028764])
+
+link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-virtual-machines-cli_mtv[MTV-333] - Convert u440fx to q35 +
+i440fx chpset with PIOS is reverted to q35 bios.
```
Comment (ahadas) [inline documentation/modules/rn-2.4.adoc:73]:
```suggestion
i440fx chipset is converted to q35 chipset.
```

File: documentation/modules/compatibility-guidelines.adoc
Comment (ahadas) [inline documentation/modules/compatibility-guidelines.adoc:1]: did you intend to include other files besides this one in this PR?

**Review by ahadas — COMMENTED** [review]
@HagayVider1 please check out [this link](https://github.com/apps/dco) that describes how to "pass" DCO

**Review by ahadas — APPROVED** [review]

**Review by ahadas — COMMENTED** [review]
we should also update the version of Openshift in the "Installing the MTV Operator" section

**Comment by HagayVider1** [comment]
@ahadas  @bkhizgiy @anarnold
Updated Compatible software versions table.
Added OpenStack column.

**Review by ahadas — APPROVED** [review]
the versions look right, there are other changes that should drop/change when rebasing the PR

**Comment by anarnold97** [comment]
Closed – obsolete

**Pull request closed without merging**